src/django/api/models.py

```diff
@@ -167,6 +173,12 @@ class Contributor(models.Model):
         on_delete=models.SET_NULL,
         null=True,
         help_text=('The embedded map configuration for the contributor'))
+    embed_level = models.IntegerField(
+        null=False,
+        blank=False,
+        default=1,
```

Comment:
We are charging for access to the embedded map. The way I had envisioned implementing that was to make this `embed_level` field nullable and have the default null value represent no embed access.

A follow up feature to this PR will be to change the content of the "Embed" tab to a "contact us to purchase embedded map" message.